packages/palette/package.json

```diff
@@ -123,6 +123,7 @@
     "babel-plugin-styled-components": "^1.10.0",
     "d3-interpolate": "^1.3.2",
     "d3-shape": "^1.3.5",
+    "debounce": "^1.2.0",
```

Comment:
It looks like debounce is just hacking around a bug in the RadioButton but I haven't really looked at it. This package is tiny enough as a replacement for now.